src/api/app/helpers/webui/patchinfo_helper.rb

```diff
@@ -4,4 +4,27 @@ def patchinfo_bread_crumb(*args)
     args.insert(0, link_to(@package, package_show_path(project: @project, package: @package)))
     project_bread_crumb(*args)
   end
+
+  def patchinfo_header(patchinfo, package_names)
+    list = package_names.to_sentence
+    text = "#{patchinfo.category} update for #{truncate(list, length: 60)}"
```

Comment:
I believe the category should be displayed just like the rating is, so with a badge.